.buildkite/release-builds.yml

```diff
@@ -0,0 +1,39 @@
+# This script is run via Buildkite's scheduled jobs feature.
+#
+# It's meant to rebuild various CI caches on a periodic async basis, so as
+# not to waste time on every CI job updating the cache.
+
```

Comment:
There might be some copy-paste leftover here, but I'm not sure at which level.

If this pipeline is meant to run on a schedule then I would suggest updating the explanation comment to something like:

```suggestion
# This script is run via Buildkite's scheduled jobs feature.
#
# It's meant to build and deploy various installable builds on a periodic
# async basis.

```

But, I checked the current schedules, and there's none:

![image](https://user-images.githubusercontent.com/1218433/152161498-18ac9647-ea14-40c6-8849-d93a456c133b.png)

That could just be because this PR hasn't been merged yet, and configuring before it's merged it schedule would result in failed builds. But it also made me wonder whether this pipeline is actually meant to run on a schedule at all? If that's not the case, then I'd simply remove the whole comment and trust that a reader would gather what this is about based on the content and name alone.

```suggestion
```

Reply:
Yikes – just some major copy/paste blindness 🤦 

In this case it's meant to be called exclusively via the API, which I've noted in 45cf8a5505c7a8425d3101780d34fc68373e54e9